client/my-sites/stats/stats-module-utm/utm-export-button.jsx

```diff
@@ -0,0 +1,69 @@
+import { Button, Gridicon } from '@automattic/components';
+import { saveAs } from 'browser-filesaver';
+import { useTranslate } from 'i18n-calypso';
+
+function UTMExportButton( { data } ) {
+	// Button set up.
+	const translate = useTranslate();
+	const buttonLabel = translate( 'Download data as CSV' );
+	const shouldDisableExport = data && data.length !== 0 ? false : true;
+
+	// Turns the working data into a flattened array of objects.
+	// Preserves the original data but updates the label for export.
+	const flattenDataForExport = ( originalData ) => {
+		const newData = [];
+		// Map the array of objects.
+		originalData.forEach( ( row ) => {
+			// Wrap label in quotes and push values.
+			let newLabel = `"${ row.label }"`;
```

Comment:
Just to be safe, we probably want to deal with any double quotes in `row.label` before quoting it, example:

https://github.com/Automattic/wp-calypso/blob/ae401a84d6e9fe28cefc9dc4fd3045f4891d1746/client/my-sites/stats/stats-download-csv/index.jsx#L51

Also I feel like the logic to update the label belongs to `prepareDataForDownload` more.

Reply:
Both bits of feedback have been addressed in #88708. #:+1: